spec/features/expenses/views_show_page.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+feature "view single expense" do
+  scenario "successfully" do
+    user = create(:user)
+    create(:budget, user: user)
+    expense = create(:expense)
```

Comment:
Again, this expense is not associated with the user's budget. Which means that any user can see any expense.

We should add a controller spec, to be redirected back to my expenses index view, when I try to view someone else's expense.

Reply:
Good pick up and good idea, done in sha: 5393979